microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -138,7 +138,7 @@ def verify_dpdk_ovs(
 
         try:
             # run OVS tests, providing OVS with the NIC info needed for DPDK init
-            ovs.setup_ovs(node.nics.get_nic_by_index().pci_slot)
+            ovs.setup_ovs(node.nics.get_nic_by_index(1).pci_slot)
```

Comment:
does `1` mean first non-primary nic? A well named method can help it clearer.

Reply:
Good point! Added a commit to make some clearer getter functions, get_primary and get_secondary (see updates)